teamcity/changelog.d/18041.fixed

```diff
@@ -0,0 +1 @@
+[AGENT-11701] Fix handling of projects with no builds
```

Comment:
```suggestion
Fix handling of projects with no builds. We used to refresh all the projects whenever we encountered a build config that didn't have any builds associated with it. Now we refresh only the specific build config that's lacking builds.
```

The target audience for these changelog snippets is users, folks who don't have access to Jira and typically care about the behavior more than implementation.